web/src/features/charts/ChartTitle.tsx

```diff
@@ -8,11 +8,10 @@ type Props = {
   translationKey: string;
   hasLink?: boolean;
   badgeText?: string;
-  icon?: JSX.Element;
   unit?: string;
 };
 
-export function ChartTitle({ translationKey, badgeText = undefined, icon, unit }: Props) {
```

Comment:
If we keep the icon files, should we keep this prop as well? Or do you mean that we might use the icons elsewhere in the future?

Reply:
We might use them elsewhere in the future. I don't think we will be using them inside the titles though  so it's fine to remove the prop but @Alportan can confirm.

Reply:
We won't be using them inside titles so all good 👌